doc/source/whatsnew/v0.21.0.txt

```diff
@@ -716,6 +716,7 @@ Deprecations
 
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
 - :func:`read_excel()` has deprecated ``parse_cols`` in favor of ``usecols`` for consistency with :func:`read_csv` (:issue:`4988`)
+- :func:`read_csv()` has deprecated the ``tupleize_cols`` argument (:issue:`17060`)
```

Comment:
you can say that this will always convert to a MI

Reply:
Yep, done.